cartridges/cloudinary_sg_changes/cartridge/templates/default/search/productnav.isml

```diff
@@ -164,7 +165,7 @@
 						<img alt="${nextProdUrls.imgAlt}"
 							title="${nextProdUrls.imgTitle}"
 							src="${nextProdUrls.imgUrl}"
```

Comment:
please have src only for nonresponsive, and data-src only for responsive. Also, use the standard condition